src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -103,6 +103,10 @@ class EulerDiscreteScheduler(SchedulerMixin, ConfigMixin):
         interpolation_type (`str`, default `"linear"`, optional):
             interpolation type to compute intermediate sigmas for the scheduler denoising steps. Should be one of
             [`"linear"`, `"log_linear"`].
+        use_karras_sigmas (`bool`, *optional*, defaults to `False`):
+            Use karras sigmas. For example, specifying `sample_dpmpp_2m` to `set_scheduler` will be equivalent to
+            `DPM++2M` in stable-diffusion-webui. On top of that, setting this option to True will make it `DPM++2M
+            Karras`.
```

Comment:
Could we also include the reference paper that introduced it?

https://github.com/huggingface/diffusers/pull/2874#discussion_r1154578720

Feel free to also include it in `src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_k_diffusion.py`.

Reply:
Fixed.